File: TP/c9/Exam.java
/*
#school - ТУЕС www.elsys-bg.org
#class - 11Б
#num - 20
#name - Красимир Светославов Стойков
#task - Програмата взима име на csv файл от подадените й аргументи и събира числата от втората колонка от него. Извежда се сумата.
*/
import java.io.*;
public class Exam {
	public static void main(String[] args){
		String fileName=null;
		if (args.length > 0) {
			fileName=args[0];
		} else {
			System.out.println("Wrong filename");
		}
		int sum = 0;
		try {
			BufferedReader reader = new BufferedReader(new FileReader(fileName));
			String line;
			while((line = reader.readLine())!=null) {
				sum+=Integer.parseInt(line.split(",")[1]);
			}
			System.out.println(sum);
		} catch(Exception e) {
		}
	}
}

File: TP/c9/exam.rb
#school - ТУЕС www.elsys-bg.org
#class - 11Б
#num - 20
#name - Красимир Светославов Стойков
#task - Програмата взима име на csv файл от подадените й аргументи и го отваря. В нов файл се записват резултатите от делението на числата в файла, от който четем, с две. Намира се сумата на числата от втората колонка на файла, от който четем. Сумата се извежда.
require 'csv'
input_file = ARGV[0]
p input_file

output_file = "output" + input_file
puts output_file
sum=0
CSV.open(output_file, "w") do |csv|
	CSV.foreach(input_file) do |row|
		csv << [row[0].to_i/2, row[1].to_i/2]
		sum += row[1].to_i
	end
end
p sum



